### ISSUE TITLE
Store URL errors with "Invariant Violation"

### ISSUE THREAD

**Issue by amp9020** [cezerin/cezerin#512]
Hello,

I pulled the latest version from git and followed the instructions.
When starting the app, I get the following error in the console.

ubuntu@ip-10-0-0-206:~/app/cezerin$ npm start

> cezerin@0.33.0 start /home/ubuntu/app/cezerin
> concurrently npm:start-*

[api]
[api] > cezerin@0.33.0 start-api /home/ubuntu/app/cezerin
[api] > node src/api/server/index.js
[api]
[store]
[store] > cezerin@0.33.0 start-store /home/ubuntu/app/cezerin
[store] > node dist/store/server/index.js
[store]
[store] info: Store running at http://localhost:3000
[api] info: API running at http://localhost:3001
[api] info: MongoDB connected successfully
**[store] Warning: Failed prop type: The prop `to` is marked as required in `NavLink`, but its value is `undefined`.
[store]     in NavLink
[store] Warning: Failed prop type: The prop `to` is marked as required in `Link`, but its value is `undefined`.**
[store]     in Link
[store]     in Route
[store]     in NavLink
[store]     in div
[store]     in li
[store]     in HeadMenuItem
[store]     in ul
[store]     in HeadMenu
[store]     in div
[store]     in div
[store]     in header
[store]     in Unknown
[store]     in Header
[store]     in Unknown
[store]     in SharedContainer
[store]     in Connect(SharedContainer)
[store]     in Route
[store]     in withRouter(Connect(SharedContainer))
[store]     in App
[store]     in Router
[store]     in StaticRouter
[store]     in Provider
[store] error: Page error
^C[api] npm run start-api exited with code null
[store] npm run start-store exited with code null

When visiting the localhost / store URL , i get this message:
**{"name":"Invariant Violation","framesToPop":1}**

Any clues on what is causing this, would be much appreciated.
Let me know if you need any other details.

Thanks in advance,

**Comment by sonufrienko** [comment]
@amp9020 
- *Linux*
- *Node v8.11.3*
- *pm2 v3.0.0*

1) npm i
2) npm run build
3) pm2 start process.json

No error.

**Comment by sonufrienko** [comment]
Hi @amp9020 
Whats your environment and versions?
Do you run `npm i` and `npm run build`?

**Comment by amp9020** [comment]
hello @sonufrienko 

Using aws - Ubuntu 16.04.4 LTS

node -v 
v10.6.0

pm2 -v
3.0.0

I used the getting started - github instructions and followed them.  Yes, I run npm i and npm run build .
I do use pm2 but to get the message in the previous message, I ran npm start.

The admin interface and api work perfectly fine.  I even tested the api and it is functional.

The messages are :
[store] Warning: Failed prop type: The prop to is marked as required in NavLink, but its value is undefined.
[store] in NavLink
[store] Warning: Failed prop type: The prop to is marked as required in Link, but its value is undefined.

Is this something I can adjust from the admin side?  Or do you think I can change something in a config file to accommodate that error?

FOr fun, I'll just start all over.
Let me try again and start over.  Any feedback on those messages would be greatly appreciated.

Thanks,
amp9020

**Comment by amp9020** [comment]
Balls.
Greedy me didn't read.  I forgot to run npm setup (db indexes and such).
Works now.  THanks! Closing ticket!
